src/server/scripts/Commands/cs_pet.cpp

```diff
@@ -69,10 +69,10 @@ class pet_commandscript : public CommandScript
         }
 
         CreatureTemplate const* creatureTemplate = creatureTarget->GetCreatureTemplate();
-        // Creatures with family 0 crashes the server
-        if (!creatureTemplate->family)
+        // Creatures with family CREATURE_FAMILY_NONE crashes the server
+        if (creatureTemplate->family == CREATURE_FAMILY_NONE)
         {
-            handler->PSendSysMessage("This creature cannot be tamed. (family id: 0).");
+            handler->PSendSysMessage("This creature cannot be tamed. (family id: CREATURE_FAMILY_NONE).");
```

Comment:
maybe it's useful to log both values in this case since the database doesn't store "CREATURE_FAMILY_NONE"